v1.1/pvp-specification.md

```diff
@@ -100,6 +100,10 @@ name is taken from another package (e.g. when `network-bytestring` was merged
 into `network`) or is quite general (`Data.Set` or something similar) then the
 version increase **SHOULD** be major.
 
+1. *Non-enumerated changes*. As the Haskell language evolves, new features may be introduced (e.g. rewrite rules)
+which are not explicitly covered in the above rules. In such cases, in keeping with the spirit of the PVP, any changes not
+enumerated above which would cause a package depending on the versioned package to fail to compile **MUST** be considered breaking.
```

Comment:
Thinking about this a bit more, I'm not sure this is quite right. As it stands, exporting new names is a "non-breaking change", but this _can_ cause downstream packages to not compile if they don't use explicit import lists. I think perhaps the statement we want is "unavoidably cause a downstream package to fail to compile"? I'm not sure.

Reply:
Hrm. I think that the export of names is a sort of exception carved out on a one-off basis after a bunch of discussion (and it was controversial). This clause isn't supposed to describe anything explicitly enumerated -- its the default policy for stuff that is not otherwise covered, and there I think its better to be on the safe side, allowing for exceptions to be made only after discussion of the trade-offs.